app/controllers/api/v1/events/nearests_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::Events::NearestsController < ApiController
+  def index
+    @events = Event.near(
+      [params[:lat], params[:lon]],
+      params[:radius],
+      units: :km
+    )
+    if @events.count(:all) > 0
```

Comment:
Instead of making a SQL call for the table row count, can rely on `presence` so

`@events.present?` will make sure that the collect is not blank. which means not nil and not empty

1 less SQL call is always a good thing however minor it is